### PR TITLE
add undecided unity-folder-structure ADR

### PR DESCRIPTION
**Pull request by cdingm**
this ADR needs to be decided.

### CODE REVIEW

**Comment by cdingm** [comment]
the md file will be renamed with the date, once its decided

File: adr/unity-folder-structure.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Unity Folder Structure and File names
```
Comment (delvh) [inline adr/unity-folder-structure.md:1]:
```suggestion
# Unity Folder Structure and Filenames
```

File: adr/unity-folder-structure.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Currently our Folder and File structure in our unity project is not consistent.
```
Comment (delvh) [inline adr/unity-folder-structure.md:3]:
```suggestion
Currently, the File and Folder structure in our unity projects is inconsistent.
```

File: adr/unity-folder-structure.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+There are Scripts in Scenes Folders, Scenes in no folder at all, etc. Also the Filenames do not follow a same Text Case.
```
Comment (delvh) [inline adr/unity-folder-structure.md:4]:
```suggestion
There are Scripts in Scenes Folders, Scenes in no folder at all,... Additionally, the Filenames do not follow a common naming scheme.
```

File: adr/unity-folder-structure.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+## Possible Solutions
+
+- Change nothing
+- Only put files in the correct folders, which are correctly named.
+  - for example put all World Scenes into the worlds folder
+  - Scripts only in the scripts folder, with subfolders which are named to display where the script belongs to
+    - also applies to all other stuff like prefabs, scenes, sprites, tiles, animations, ...
+  - no longer needed scenes/scripts/etc. should be moved to "old" folder in the respective parent Folder.
+    - or deleted?
```
Comment (delvh) [inline adr/unity-folder-structure.md:14]: deleted.
Why should we keep outdated states?
That's what Git is for.

File: adr/unity-folder-structure.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+Currently our Folder and File structure in our unity project is not consistent.
+There are Scripts in Scenes Folders, Scenes in no folder at all, etc. Also the Filenames do not follow a same Text Case.
+
+## Possible Solutions
+
+- Change nothing
+- Only put files in the correct folders, which are correctly named.
+  - for example put all World Scenes into the worlds folder
+  - Scripts only in the scripts folder, with subfolders which are named to display where the script belongs to
+    - also applies to all other stuff like prefabs, scenes, sprites, tiles, animations, ...
```
Comment (delvh) [inline adr/unity-folder-structure.md:12]: I think you should explicitly mention for everything where it should be put.

**Review by delvh — COMMENTED** [review]
I vote in favor of Unifying the filestructure.

**Review by Aaron2000119 — APPROVED** [review]

**Review by LeviUniAcc — APPROVED** [review]

**Review by delvh — CHANGES_REQUESTED** [review]

File: adr/unity-folder-structure.md (hunk 4 above)
Comment (cdingm) [reply adr/unity-folder-structure.md:14]: yes because nobody said anything

Comment (cdingm) [reply adr/unity-folder-structure.md]: fixed now

**Pull request merged**